equinix/resource_metal_device_acc_test.go

```diff
@@ -938,6 +916,30 @@ resource "equinix_metal_device" "test_ipxe_missing" {
   always_pxe       = true
 }`
 
+func testAccMetalDeviceConfig_timeout(projSuffix string) string {
+	return fmt.Sprintf(`
+resource "equinix_metal_project" "test" {
+    name = "tfacc-device-%s"
+}
+
+resource "equinix_metal_device" "test" {
+  hostname         = "tfacc-test-device"
+  description      = "test-desc"
+  plan             = "c3.small.x86"
+  metro            = "DA"
```

Comment:
For end-to-end tests, the plan, metro, and OS should be parameterized so that the tests are more flexible:
- see https://github.com/equinix/terraform-provider-equinix/pull/364/files#diff-b5bc3c33d7df2c76175fac51f0b7b4a5d4e2fd0268430d3b28091961885d775fL761-L763 for an example
- and note that some extra stuff needs to be added in to the test config to provide the values for those fields: https://github.com/equinix/terraform-provider-equinix/pull/364/files#diff-b5bc3c33d7df2c76175fac51f0b7b4a5d4e2fd0268430d3b28091961885d775fL771